app-template/package-template.json

```diff
@@ -59,7 +59,7 @@
     "bezier-easing": "^2.0.3",
     "bhttp": "1.2.1",
     "bitauth": "^0.2.1",
-    "bitcore-wallet-client": "https://github.com/Bitcoin-com/bitcore-wallet-client.git",
+    "bitcore-wallet-client": "https://github.com/vinarmani/bitcore-wallet-client.git",
```

Comment:
Please, change to Bitcoin-com and I will merge to our task branch. Then we will manage this part in our side until production.

Reply:
Done